services/api/service.go

```diff
@@ -381,6 +398,46 @@ func (api *RelayAPI) startValidatorRegistrationDBProcessor() {
 	}
 }
 
+// simulateBlock sends a request for a block simulation to blockSimRateLimiter.
+func (api *RelayAPI) simulateBlock(opts blockSimOptions) error {
+	t := time.Now()
+	simErr := api.blockSimRateLimiter.send(opts.ctx, opts.req, opts.highPrio)
+	if simErr != nil {
+		log := opts.log.WithField("simErr", simErr.Error())
+		log.WithError(simErr).WithFields(logrus.Fields{
```

Comment:
Consider having a variable for the `logrus.Fields` object to avoid repeating 3 lines.

Reply:
done!